validator_client/Cargo.toml

```diff
@@ -62,3 +62,4 @@ malloc_utils = { workspace = true }
 sysinfo = { workspace = true }
 system_health = { path = "../common/system_health" }
 logging = { workspace = true }
+fdlimit = "0.2.1"
```

Comment:
Can you use version 0.3.0? It includes an improvement to not panic on syscall errors, https://github.com/paritytech/fdlimit/commit/eee618fbe779452099f1857dd9191ec711067af6